### PR TITLE
Ticket 5: get article

### PR DESCRIPTION
**Pull request by omobayed**

### CODE REVIEW

File: src/components/Article.jsx [hunk 1]
```diff
@@ -6,20 +6,34 @@ import Moment from 'moment';
...
     const [article, setArticle] = useState([])
-    
-  
+    const [isLoading, setIsLoading] = useState(false);
+    const [isError, setIsError] = useState(false);
 
     const { article_id } = useParams()
 
     useEffect(() => {
+        setIsLoading(true);
```
Comment (Beth-McCormack) [inline src/components/Article.jsx]: You can set the isLoading to be true on line 9 which then means you dont need this line🙂

File: src/components/Articles.jsx [hunk 2]
```diff
@@ -4,19 +4,31 @@ import ArticleCard from './ArticleCard'
...
 
     if (isLoading)
-        return <p>loading...</p>
+        return <article className="articles-cards"><p>Loading articles...</p></article>
+
+    if (isError)
+        return <article className="articles-cards"><p>Error while loading all articles...</p></article>
 
     return (
         <article className="articles-cards">
```
Comment (Beth-McCormack) [inline src/components/Articles.jsx:34]: Please make sure the img tag has an "alt" attribute for accessibility🙂

**Review by Beth-McCormack — APPROVED** [review]
Nice work! I've left a couple of comments, please implement these before merging then feel free to move onto the next ticket🙂

File: src/components/Articles.jsx (hunk 2 above)
Comment (omobayed) [reply src/components/Articles.jsx:34]: done

**Pull request merged**